packages/cli/package.json

```diff
@@ -69,11 +69,11 @@
     ]
   },
   "dependencies": {
-    "@ceramicnetwork/3id-did-resolver": "^2.19.0",
-    "@ceramicnetwork/http-client": "^2.23.0",
-    "@ceramicnetwork/stream-model": "^1.8.0",
-    "@ceramicnetwork/stream-model-instance": "^1.8.0",
-    "@ceramicnetwork/streamid": "^2.14.0",
+    "@ceramicnetwork/3id-did-resolver": "2.23.0-rc.0",
```

Comment:
why pinning to specific versions?  We should be able to keep the `^` here and be guaranteed to get a version at *least* after that RC, but still be able to pick up newer versions as they get released

Reply:
Since this is also for an RC, we want to know exactly what versions are being for things that may change, e.g. js-ceramic, so we can more easily address bugs or know if a bug has been fixed and this RC just has out of date versions.

Reply:
I don't understand why we wouldn't want to automatically pick up the fully released version once the RC is promoted to full release

Reply:
https://linear.app/3boxlabs/issue/CDB-2618/determine-appropriate-strategy-for-using-js-ceramic-rcs-in-js